src/Dashboard/Dashboard.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import Dashboard from "./Dashboard";
+import "@testing-library/jest-dom";
+import { render, waitFor } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import { getRandomCocktail } from "../apiCalls";
+import { mocked } from "ts-jest/utils";
+jest.mock("../apiCalls");
+
+describe("Dashboard", () => {
+  mocked(getRandomCocktail).mockImplementation(() =>
```

Comment:
is mockImplementation a keywork for TS Jest?

Reply:
That's how I understood it

Reply:
Very cool!